content/docs/api-reference/open.md

```diff
@@ -199,3 +205,25 @@ import dvc.api
 with dvc.api.open('data/nlp/words_ru.txt', encoding='koi8_r') as f:
     # ... Process Russian words
 ```
+
+## Example: Specify credentials for your remote
+
+See [remote modify](/doc/command-reference/remote/modify) for full list of
+remote-specific config options.
+
+```py
+import dvc.api
+
+config = {
+    'remote': {
+        'myremote': {
+            'access_key_id': 'mykey',
+            'secret_access_key': 'mysecretkey',
+            'session_token': 'mytoken',
+        },
+    },
+}
```

Comment:
If there are other remotes configured, including a default, which one prevails?

Reply:
@dberenbaum This example assumes you have `myremote` set as default. You could change the default yourself through this too (config["core"]["remote"] = myremote).

Reply:
Are there other config options besides `remote` that will ever make sense to set?

Reply:
My worry is that people may literally copy this and reuse the `myremote` name and then be confused when it has no effect because that doesn't match the name of the default remote.

Reply:
Thinking about this, not sure how real is `myremote` concern. It is literally saying `myremote`, which implies that you need to put in the remote name that you use. Maybe what would help is an additional example for setting a default remote as well.